### ISSUE TITLE
Documentation for calendar-events to include ranges + repititon value

### ISSUE THREAD

**Issue by eikek** [systemd/systemd#15030]
**Is your feature request related to a problem? Please describe.**
I'm trying to understand the syntax and semantics of the repetitive calendar events. The man page `systemd.time` has good info, but I'm missing a clear description on what it means to put a repetition value to a range. My first guess was that the repetition applies to start and end values just as with a single value, generating multiple ranges. So `1..3/5` → `1..3,6..8,11..13,…`. Trying to figure out more using `systemd-analyze` did not help, unfortunately.

I'm using systemd 243. Many calendar events I tried are not valid, according to `systemd-analyze`, for example:
```
> systemd-analyze calendar --iterations=5 "*-*-* *:1..5/5:0"
Failed to parse calendar specification '*-*-* *:1..5/5:0': Invalid argument
```
A slight variation then works:
```
> systemd-analyze calendar --iterations=5 "*-*-* *:1..6/5:0"
  Original form: *-*-* *:1..6/5:0
Normalized form: *-*-* *:01..06/5:00
    Next elapse: Fri 2020-03-06 00:01:00 CET
       (in UTC): Thu 2020-03-05 23:01:00 UTC
       From now: 12min left
       Iter. #2: Fri 2020-03-06 00:06:00 CET
       (in UTC): Thu 2020-03-05 23:06:00 UTC
       From now: 17min left
       Iter. #3: Fri 2020-03-06 01:01:00 CET
       (in UTC): Fri 2020-03-06 00:01:00 UTC
       From now: 1h 12min left
       Iter. #4: Fri 2020-03-06 01:06:00 CET
       (in UTC): Fri 2020-03-06 00:06:00 UTC
       From now: 1h 17min left
       Iter. #5: Fri 2020-03-06 02:01:00 CET
       (in UTC): Fri 2020-03-06 01:01:00 UTC
       From now: 2h 12min left
```
I don't understand why that is. And looking at the iterations, this seems the same as `*-*-* *:1,6:0`. I couldn't find more information than what the man page shows. I hope I didn't miss anything; if not, maybe a update to the man page would be a good idea?

**Describe the solution you'd like**
Extend the man page by a few more sentences or just a few examples related to ranges + repetition value.

**Describe alternatives you've considered**
–

**Comment by poettering** [comment]
1..17/5 means "every 5 seconds in the range 1..17. Which means on 1, on 1+5, on 1+10, on 1+15. But not on 1+20 anymore, since that's above 17...

**Comment by eikek** [comment]
Thank you! Now that I see it, I'm wondering why this didn't occur to me…

**Comment by poettering** [comment]
Fix in #15645 

(i also made sure your first expression parses now, and is normalized so that the repetition just goes away)
